juju/python-libjuju
# chore: add juju 3.1.{3,4,5,6,7,8,9,10} schemas, no code changes

#### Description

After every Juju 3.X.Y release, python-libjuju should include Juju's `apiserver/facades/schema.json` file under `juju/client/schemas-juju-3.X.Y.json`, regenerate the client code, and verify whether any new facades are supported. The latest schemas currently in python-libjuju are 3.1.2 for the 3.1 series, and 3.3.0 for the 3.3 series, so clearly this hasn't been done in a while.

This PR adds (client-only) schemas for 3.1.3 - 3.1.10. There are no facade version changes, so despite regenerating the python-libjuju client code, this PR makes no changes other than adding the schema files themselves.

While it seems kind of wasteful to add these large schema files when they're not making any facade version changes, I think it makes sense to do so as a clear signal that python-libjuju is up to date when it comes to supporting the 3.1 series.

#### QA Steps

After adding each schema we run:
rm juju/client/_client*.py juju/client/_definitions.py followed by:
make client
and verify that there are no changes to the generated code.
